### ISSUE TITLE
Generated examples don't compile with r2.4

### ISSUE THREAD

**Issue by cpetig** [cpetig/tflite_micro_compiler#59]
I just found that Register_CONV_2D,QUANTIZE,FULLY_CONNECTED,SOFTMAX,DEPTHWISE_CONV_2D are not declared in micro_ops.h
I need to figure out how to fix the examples!

PS: Sorry, I did not fall off the edge of the world, I just spend most of my time on gitlab with the Rust written game veloren.net

**Comment by PhilippvK** [comment]
I fixed my generated model manually by replacing the prefix `tflite::ops::micro::` with ``tflite::` for the following registration methods which have been moved to a flat namespace:
  - `Register_CONV_2D`
  - `Register_QUANTIZE`
  - `Register_SHAPE`
  - `Register_SOFTMAX`
  - `Register_SVDF`
  - `Register_FULLY_CONNECTED`

**Comment by PhilippvK** [comment]
I also had to manually add `#include "tensorflow/lite/micro/kernels/fully_connected.h` to my models since October.

**Comment by cpetig** [comment]
Thank you for sharing this information, this saves me a lot of time.
But I really enjoyed needing no longer any operator specific code for half a year. :disappointed:

**Comment by andrewstevens-infineon** [comment]
Hi Guys,

Sorry - I actually have a maintained version I maintain here (Infineon)  for our deployment toolchain project with relevant patches and some modest extensions.   I just haven't got around to pushing it upstream recently due to project-pressure (I'd need to check everythong works cleanly for a stock tflite(u) kernel.    We're just migrating to 2.4.1 as our baseline so this would be a good time to do the necessary and push a PR for Christoph.
